### PR TITLE
docs: add link update alert to README.md

### PR DESCRIPTION
**Pull request by bernhardreiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@
  Software-Engineering: 2024 Intevation GmbH <https://intevation.de>
 -->
 
+
+> [!IMPORTANT]
+> Adjust your links and go module paths if you still have `csaf-poc` in them, to avoid future breakage.
+> (This repository was moved here on 2024-10-28. The old one is decrecated
+> and redirection will be switched off in a few months.)
```
Comment (tschmidtb51) [inline README.md:15]:
```suggestion
> and redirection will be switched off a few months later.)

**Review by tschmidtb51 — APPROVED** [review]
IMHO, that sounded like a moving target as it was pointing into the indistinct, distant future

File: README.md (hunk 1 above)
Comment (bernhardreiter) [reply README.md]: changed

**Review by tschmidtb51 — DISMISSED** [review]
LGTM

**Review by JanHoefelmeyer — APPROVED** [review]
LGTM

**Review by tschmidtb51 — APPROVED** [review]
LGTM

**Pull request merged**